variables-skyline/index120.html

```diff
@@ -67,5 +67,5 @@
 </html>
 
 <!--
-  I don't really like that color. Change the value of your variable from `#999` to `#aa80ff` and you can see how it gets applied everywhere you used the variable.
+  I don't really like that color. Change the value of your variable from `#999` to `#aa80ff` and you can see how it gets applied everywhere you used the variable. This is the main advantage of using variables, being able to quickly change many values in your stylesheet by just changing the value of a variable.
```

Comment:
Made a slight change: "I don't really like that color. Change the value of your variable from `#999` to `#aa80ff` and you can see how it gets applied everywhere you used the variable. This is the main advantage of using variables -- being able to quickly change many values in your stylesheet by just changing a single variable."